internal/codegen/golang/postgresql_type.go

```diff
@@ -179,16 +245,21 @@ func postgresType(req *plugin.CodeGenRequest, col *plugin.Column) string {
 
 	case "cidr":
 		switch driver {
+		case SQLDriverPGXV5:
+			if notNull {
+				return "netip.Prefix"
+			}
+			return "*netip.Prefix"
 		case SQLDriverPGXV4:
-			return "pgtype.CIDR"
-		case SQLDriverLibPQ:
```

Comment:
Removing this cases the build to break. Can we add it back in?

Reply:
Fixed